packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -9,17 +9,13 @@ import { global_BackgroundColor_100 as globalBackgroundColor100 } from '@pattern
 
 class ComplianceRemediationButton extends React.Component {
     formatRule = ({ title, refId }, profile, system) => ({
-        id: `ssg:rhel7|${profile}|${refId}`,
+        id: `ssg:rhel7|${profile.split('xccdf_org.ssgproject.')[1]}|${refId}`,
```

Comment:
When this has some garbled up profile refId, it will return undefined - which is fine, because Remediations returns a default profile for that